plugins/jira/commands/backlog.md

```
@@ -0,0 +1,599 @@
+---
+description: Find suitable JIRA tickets from the backlog to work on based on priority and activity
+argument-hint: [project-key] [--assignee username] [--days-inactive N]
+---
+
+## Name
+jira:backlog
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)